package.json

```diff
@@ -52,9 +52,8 @@
   },
   "scripts": {
     "browserstack": "sh build/run-browserstack.sh",
-    "ci": "grunt test",
-    "ci-include-lint": "grunt eslint && grunt test",
-    "test": "grunt",
+    "build": "grunt build",
+    "test": "grunt test",
```

Comment:
Did not realize it was deprecated. Looks like it happened in npm 4. We could change it to the new `npm prepare` hook, but I think just including the build in the `test` command makes the most sense. Will update here shortly.